### PR TITLE
[IMP] sales: updated ebay configuration doc

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3590260&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### FWP up to `master`

### CODE REVIEW

**Comment by ksc-odoo** [comment]
@dalaOdoo if you could also give this a quick review when you get a chance, that would be great! Thanks! 👍

File: content/applications/sales/sales/ebay_connector/setup.rst
Comment (jero-odoo) [inline content/applications/sales/sales/ebay_connector/setup.rst:24]:
```suggestion
Then, click :guilabel:`Create a keyset` beneath, and proceed to enter in the necessary information
```

File: content/applications/sales/sales/ebay_connector/setup.rst
Comment (jero-odoo) [inline content/applications/sales/sales/ebay_connector/setup.rst:12]: I would consider changing this up just to give us some plausible deniability. "May take up to one business day" or something similar. (Unless you took this directly from eBay, then feel free to ignore)

File: content/applications/sales/sales/ebay_connector/setup.rst
Comment (jero-odoo) [inline content/applications/sales/sales/ebay_connector/setup.rst:151]:
```suggestion
If working with a lot of products, eBay can sometimes refuse some synchronization calls, due to
```

**Review by jero-odoo — APPROVED** [review]
@ksc-odoo Looks great! just a few comments but I think this one is ready to go. Pushing through with approval

**Comment by ksc-odoo** [comment]
Thanks @jero-odoo ! This one's ready for a second round of Peer Review, Materials Team (@hojo-odoo @Felicious @brse-odoo ) 👍

**Review by brse-odoo — COMMENTED** [review]
Hi @ksc-odoo, I've finished my peer content review of this doc, and this one is all set as well. I had very few suggestions again - mostly grammatical, for you to accept/reject as you see fit.

As always, feel free to ping me with any questions about my comments!

File: content/applications/sales/sales/ebay_connector/setup.rst
Comment (brse-odoo) [reply content/applications/sales/sales/ebay_connector/setup.rst:82]: Nope, just made it easier to read for me but not a deal breaker. It's fine as is

**Comment by ksc-odoo** [comment]
Thanks for the always-helpful feedback @brse-odoo -- once I get your approval, I'll pass this one to Final Review!

**Comment by brse-odoo** [comment]
> Thanks for the always-helpful feedback @brse-odoo -- once I get your approval, I'll pass this one to Final Review!

@ksc-odoo Changes look good, this one is ready for final review!

**Comment by ksc-odoo** [comment]
Thanks @brse-odoo -- Alrighty, @odoo/us-doc-review this one's ready for Final Review! 👍

**Comment by tiku-odoo** [comment]
Duplicate PR: Cherry-picked changes and incorporated them into: https://github.com/odoo/documentation/pull/6214

Closing out PR as https://github.com/odoo/documentation/pull/6214 has been pushed

**Pull request closed without merging**
